scripts/pc2sandbox.sh

```diff
@@ -227,32 +290,47 @@ COMMAND_EXIT_CODE=$?
 
 kills=`grep oom_kill $PC2_SANDBOX_CGROUP_PATH/memory.events | cut -d ' ' -f 2`
 
+KillChildProcs
+
+# Get cpu time used.
+cputime=`grep usage_usec $PC2_SANDBOX_CGROUP_PATH/cpu.stat | cut -d ' ' -f 2`
+
+# Get wall time - we want it as close as possible to when we fetch the cpu time so they stay close
+# since the cpu.stat includes the time this script takes AFTER the submission finishes.
+endtime=`GetTimeInMicros`
```

Comment:
With this call to `GetTimeInMicros` located here, the user is being charged for the time taken by `KillChildProcs` and also the `grep` which gets the cputime.  It seems like this call (and hence probably also the call to compute walltime) belong closer to the `wait`, above.

Reply:
See above.  Not "charged".  It is for reporting to the log file only.